plugins/org.komodo.core/src/config/vdb.cnd

```diff
@@ -107,6 +107,7 @@
 
 // permission resource path is node path
 [vdb:permission] > nt:unstructured
+ - vdb:resourceName (string) mandatory
```

Comment:
Since resource names are unique, I designed the `vdb:permission` node type name to be the resource name. Do you think this will still work? If we do add a new property for the resource name, what will the node name be? And we can't forget that if we do make a change to this CND, we need to also change it in ModeShape codebase, as well as changing the ModeShape sequencer.